packages/eslint-plugin/src/rules/dependency-checks.ts

```diff
@@ -40,7 +40,7 @@ export default createESLintRule<Options, MessageIds>({
     type: 'suggestion',
     docs: {
       description: `Checks dependencies in project's package.json for version mismatches`,
-      recommended: 'error',
+      recommended: 'recommended',
```

Comment:
Should we add migrations here as a followup?

Reply:
Yes